### PR TITLE
Add function that checks to see if word is valid

### PR DESCRIPTION
**Pull request by javepaid**
Jira Ticket: https://lipsumtext.atlassian.net/browse/LS-5

Title is self-explanatory. Added as per request.

### CODE REVIEW

File: test/sample_test.js [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
-    it('should return -1 when the value is not present', function () {
-      assert.equal([1, 2, 3].indexOf(4), -1);
-    });
-  });
-});
+var assert = require('assert')
+
+var index = require('../src/index')
+
+describe('#checkIfWord()', function() {
```
Comment (silikeite) [inline test/sample_test.js]: Shouldn't this test be on its own file? (Or rename the test to match what it's testing.)

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+var checkWord = require('check-if-word')
+words = checkWord('en')
+
+const checkIfWord = function (word) {
```
Comment (silikeite) [inline src/index.js:4]: Looks good.

As an architectural decision, are we keeping all logic functions inside index.js?

Comment (silikeite) [reply src/index.js:4]: On second check, this library seems to not be particularly performant?
image

Comment (silikeite) [reply src/index.js:4]: `check-word` https://www.npmjs.com/package/check-word seems to be much more performant on testing.
image

File: test/sample_test.js (hunk 1 above)
Comment (javepaid) [reply test/sample_test.js]: Will work on this, thanks!

File: src/index.js (hunk 2 above)
Comment (javepaid) [reply src/index.js:4]: I honestly thought so too haha. Thanks for providing an alternative.

Comment (javepaid) [reply src/index.js:4]: > Looks good.
> 
> As an architectural decision, are we keeping all logic functions inside index.js?

I suppose this is something we can work on on a later date.

**Comment by javepaid** [comment]
Besides the concern regarding the architectural design of the software, all other concerns have been addressed.

**Review by silikeite — APPROVED** [review]
Looks good. Merging

**Pull request merged**